contents/sprint/2025/09-hsinchu.mdx

```diff
@@ -0,0 +1,112 @@
+---
+date: "2025-09-13 10:00"
+eventTime: "10:00 - 17:00"
+title: "scisprint 2025 Sep in Hsinchu"
```

Comment:
Please use "September" instead of the short form

```suggestion
title: "scisprint 2025 September in Hsinchu"
```

Reply:
I don't like capitalized "s" in sciwork, but it is fine as long as the team reach a consensus.